plugins/balancedLevelProvider.go

```diff
@@ -93,6 +108,26 @@ func validateSpread(spread float64) {
 
 // GetLevels impl.
 func (p *balancedLevelProvider) GetLevels(maxAssetBase float64, maxAssetQuote float64) ([]api.Level, error) {
+	log.Printf("Last base balance was: %v", p.lastMaxAssetBase)
+	log.Printf("Last quote balance was: %v", p.lastMaxAssetQuote)
+
+	//Checking whether balances have changed meaningfully
+	//We have to give the balances some room to move because slightly different balance values come back from Horizon even if nothing changed
+	//Probably because floating point math is the worst
```

Comment:
we should try and address this **jitter**, adding this error buffer as a solution should only be the last resort.

we're using 7 units of precision everywhere which is also used in horizon so we should be able to fix this. Can you give me details on when we're getting different results from horizon?